libevm/VMConfig.h

```diff
@@ -54,7 +54,9 @@ namespace eth
 
 
 // set this to 2, 1, or 0 for more, less, or no tracing to cerr
-#define EVM_TRACE 0
+#ifndef EVM_TRACE
```

Comment:
In case a macro is not defined `#if` should replace it with `0`. Please remove it, unless any compiler complains about it.